tg/controllers.py

```diff
@@ -499,6 +499,43 @@ def edit_msg(self) -> None:
                     self.model.edit_message(text=text)
                     self.present_info("Message edited")
 
+    def _get_user_ids(self, is_multiple: bool = False) -> List[int]:
+        users = self.model.users.get_list_of_users()
+        _, cols = self.view.stdscr.getmaxyx()
+        limit = min(int(cols / 2), max(len(user.name) for user in users),)
+        users_out = "\n".join(
+            f"{user.id}\t{user.name:<{limit}} | {user.status}"
+            for user in sorted(users, key=lambda user: user.order)
+        )
+        cmd = config.FZF + " -n 2"
```

Comment:
let's test that `fzf` exists before running this command. if it is missing we need to present meaningful msgs in status line